packages/nouns-webapp/src/pages/Governance/index.tsx

```diff
@@ -1,20 +1,25 @@
 import { Col } from 'react-bootstrap';
 import Section from '../../layout/Section';
-import { useAllProposals } from '../../wrappers/nounsDao';
+import { useAllProposals, useProposalThreshold } from '../../wrappers/nounsDao';
 import Proposals from '../../components/Proposals';
 import classes from './Governance.module.css';
 
 const GovernancePage = () => {
   const { data: proposals } = useAllProposals();
+  const threshold = useProposalThreshold();
+  const nounsCopy = threshold === 0 ? `1 Noun` : `${threshold} Nouns`;
```

Comment:
Should be `${threshold + 1} Nouns` as the number of Nouns must be greater than the proposal threshold.